sema4ai/src/sema4ai_code/action_server.py

```diff
@@ -515,3 +520,34 @@ def package_set_changelog(
             success=False,
             message=f"Error updating the changelog for package to Control Room.\n{command_result.message or ''}",
         )
+
+    def package_metadata(self, input_dir: str, output_dir) -> ActionResult:
+        """
+        Create the Action Package metadata.json.
+
+        Args:
+            input_dir, directory to find the package.json from.
+            output_dir, directory where to create the metadata.json file.
+        """
+        output_file_path = Path(output_dir, "metadata.json")
+
+        args = [
+            "package",
+            "metadata",
+            "--input-dir",
+            input_dir,
+            "--output-file",
+            str(output_file_path),
+        ]
+
+        command_result = self._run_action_server_command(
+            args, timeout=ONE_MINUTE_S * 15
```

Comment:
I think I'd actually make the timeout 60 minutes (low end machines with bad net connection can suffer here).